apps/cowswap-frontend/src/modules/permit/state/ordersPermitStatusAtom.ts

```diff
@@ -0,0 +1,10 @@
+import { atom } from 'jotai'
+import { atomWithReset } from 'jotai/utils'
+
+import { OrdersPermitStatus } from '../types'
+
+export const ordersPermitStatusAtom = atomWithReset<OrdersPermitStatus>({})
```

Comment:
You can use atomWithPartialUpdate() here

Reply:
Hmmm, actually, I had a different plan in mind when I created the atom.
I no longer need it to be reset, so just a regular atom should do.

Unless you have another idea where `atomWithPartialUpdate` fits better?

Reply:
Now I see what you mean. That is very handy!